src/ctia/properties.clj

```diff
@@ -205,7 +205,8 @@
 
                       "ctia.store.es.event.slicing.granularity"
                       (s/enum :minute :hour :day :week :month :year)})
-   (st/optional-keys {"ctia.versions.config" s/Str})))
+   (st/optional-keys {"ctia.versions.config" s/Str})
+   (st/optional-keys {"ctia.features.disable" (s/constrained s/Str #(not (re-find #"\s" %)))})))
```

Comment:
I would use a consistent naming with the feature service
```suggestion
   (st/optional-keys {"ctia.features.disabled" (s/constrained s/Str #(not (re-find #"\s" %)))})))
```

Reply:
My idea was to indicate the intent, like "I want to disable these entities" vs. "these entities are somewhat deficient already and I need to mark them as such".